src/helpers/debug.js

```diff
@@ -0,0 +1,2 @@
+import createDebug from 'debug'
```

Comment:
If we are going to standardise on `debug` then this helper should likely be in a `@dhis2/log` library.